fili-wikipedia-example/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM druidio/example-cluster:latest
+LABEL maintainer="https://groups.google.com/forum/#!forum/fili-developers"
+
+# fili github information
+ENV GITHUB_OWNER=yahoo BRANCH=master
```

Comment:
[Docker prefers this style](https://docs.docker.com/engine/reference/builder/#env) because it slightly reduces the work to build the image. I don't think it makes a big difference here, so I can change it if you like.